code/modules/cargo/packs/medical.dm

```diff
@@ -123,11 +123,11 @@
 /datum/supply_pack/medical/surgery
 	name = "Surgical Supplies Crate"
 	desc = "Do you want to perform surgery, but don't have one of those fancy \
-		shmancy degrees? Just get started with this crate containing a medical duffelbag, \
+		shmancy degrees? Just get started with this crate containing a DeForest surgery tray, \
```

Comment:
who is DeForest

Reply:
in universe medical company that I like referencing. you can see it on the surgery tray object itself, on shipping containers, and in the common core

Reply:
here's a link to the page that talks abt em. https://github.com/tgstation/common_core/blob/master/Governments%20and%20Organisations/Corporations/Corporation%20Bible.md#deforest-medical-corporation
looking back they might've been better as interdyne, but I felt like interdyne gets all the lore. so I wanted to share the love a little